sections/04_study.md

```diff
@@ -74,7 +74,7 @@ computational restrictions.
 
 Epigenetic data, combined with deep learning, may have sufficient explanatory
 power to infer gene expression. For instance, a convolutional neural network
-applied to histone modifications, termed DeepChrome, [@tag:Singh2016_deepchrome]
+applied to histone modifications, termed DeepChrome [@tag:Singh2016_deepchrome]
```

Comment:
I think the comma should be moved, not removed: `modifications, termed DeepChrome [@tag:Singh2016_deepchrome],`